script/helpers/PantosBaseScript.s.sol

```diff
@@ -122,6 +122,7 @@ abstract contract PantosBaseScript is Script {
         // Not yet known: _chainIdToPantosBlockchainIdMapping[] = BlockchainId.SONIC; // Sonic mainnet
         _chainIdToPantosBlockchainIdMapping[64165] = BlockchainId.SONIC; // Sonic testnet
         _chainIdToPantosBlockchainIdMapping[31342] = BlockchainId.SONIC; // Local Sonic dev
+        _chainIdToPantosBlockchainIdMapping[57054] = BlockchainId.SONIC; // Sonic Blaze testnet
```

Comment:
I think we should remove the old chain ID (64165) and keep only the new one, since we should operate on a single testnet at a time. Also, the mainnet chain ID is available so we should add it. 

However, this PR is created from a fork which we won't have access to. I will close this PR and create a new one.